index.html

```diff
@@ -0,0 +1,246 @@
+<!DOCTYPE html>
```

Comment:
Minor styling issues on this page with footer, card layout (-2)
Missing Navbar links (-3)